test/test_institutions.rb

```diff
@@ -7,7 +7,7 @@ def setup
   end
 
   def test_get
-    response = @client.institutions.get(count: 3, offset: 1)
+    response = @client.institutions.get(count: 3, offset: 1, options: {products: ['transactions']})
```

Comment:
Could we add an additional test, something like `test_get_with_options` so we have both the old test case and this new one you wrote?